flaskProject/resources/motos.py

```diff
@@ -0,0 +1,77 @@
+from flask_restful import Resource, reqparse
+from models.moto_model import MotoModel
+
+
+class Moto(Resource):
+    def get(self, id):
+        try:
+            moto = MotoModel.find_by_id(id)
+            return {'moto':moto.json()}, 200
+        except:
+            return {"message": "Error Get Moto"}, 500
```

Comment:
El mensaje debería ser más aclaratorio para el usuario. Como: return {'message': "Moto with id [{}] Not Found".format(id)}, 404